Dockerfile

```diff
@@ -1,12 +1,12 @@
-# Use the latest Ubuntu image as the base
-FROM ubuntu:rolling
+# Use the latest slim Debian unstable image as the base
+# TODO: Move to stable when buster is out (sid is needed for QEMU 3.1)
+FROM debian:unstable-slim
 
 # Make sure that all packages are up to date then
-# install the base Ubuntu packages that we need for
+# install the base Debian packages that we need for
```

Comment:
This raises an interesting point. Should we keep an Ubuntu container around, in addition to the Debian one?

Reply:
I wouldn't be opposed to it, shouldn't be difficult with the Makefile setup I started (`REPO=clangbuiltlinux/ubuntu make deploy`).